moj_analytics/auth0_client.py

```diff
@@ -104,26 +104,29 @@ def create(self, resource):
     def get_all(self, resource_class, **kwargs):
         endpoint = '{}s'.format(resource_class.__name__.lower())
 
-        resources = self.request('GET', endpoint, params=kwargs)
+        defaults = {
+            'page': 0,
+            'per_page': 100,
+            'include_totals': True
+        }
 
-        if endpoint in resources:
-            resources = resources[endpoint]
+        params = defaults.copy()
```

Comment:
Kind of a Python question: Why to make a copy here?

Isn't `defaults = {...}` creating a new dictionary at every `get_all()` call?

Reply:
It's being explicit about using a copy of the `defaults` dictionary, and not modifying it. It's not really necessary here because, as you say, the defaults dictionary is created just before.